cmd/submit.go

```diff
@@ -87,6 +87,8 @@ func runSubmit(cfg config.Config, flags *pflag.FlagSet, args []string) error {
 
         %s
 
+    Please provide the path to the file you wish to submit, e.g. : exercism submit FILENAME
```

Comment:
We have a variable called `BinaryName` that contains the name of the command. You should use that as opposed to hard coding `exercism` https://github.com/exercism/cli/blob/master/cmd/submit.go#L141